packages/dds/tree/package.json

```diff
@@ -1,12 +1,12 @@
 {
 	"name": "@fluidframework/tree",
 	"version": "2.0.0-rc.1.0.0",
-	"description": "Tree",
+	"description": "Distributed tree",
 	"homepage": "https://fluidframework.com",
 	"repository": {
 		"type": "git",
 		"url": "https://github.com/microsoft/FluidFramework.git",
-		"directory": "experimental/dds/tree2"
+		"directory": "packages/dds/tree"
```

Comment:
Overlooked when moving to production directory

Reply:
We used to have a policy handler for this. I wonder what happened to it.